Source/ACE/Entity/Landblock.cs

```diff
@@ -276,7 +276,7 @@ public List<WorldObject> GetWorldObjectsInRange(WorldObject wo, float maxrange,
             return allworldobj;
         }
 
-        public List<WorldObject> GetWorldObjectsByGuid(ObjectGuid objectguid, bool neighbors)
+        private List<WorldObject> GetWorldObjectsByGuid(ObjectGuid objectguid, bool neighbors)
         {
```

Comment:
I missed that I could use the other call and search adjcency - this is something stackoverflow put it - because I did not realize that the other could do what I wanted - I used this to search the landblock plus adj.   I thought it was funky but I was concentrating on finding my damn bug and did not really think about it.